deploy/olm-catalog/test/7.11.0-1/manifests/kieapp.crd.yaml

```diff
@@ -1876,6 +1876,47 @@ spec:
                                 to generated options in JAVA_OPTS. e.g. '-Dsome.property=foo'
                               type: string
                           type: object
+                        kafka:
```

Comment:
@tchughesiv  this addition was added during the build after the rebase is it ok to add ?